sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/StringExpressionsSuite.scala

```diff
@@ -840,6 +841,62 @@ class StringExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       Literal.create(null, IntegerType), Literal.create(null, IntegerType)), null)
   }
 
+  test("ToNumber") {
```

Comment:
The `usage` part of the expressions says that some part of patterns can use locale:
```
'S': sign anchored to number (uses locale)
      'L': currency symbol (uses locale)
      'D': decimal point (uses locale)
      'G': group separator (uses locale)
```
Could you test a few locales.

Reply:
@MaxGekk Thanks for this review. I test the locale on Postgresql but 'L' seems not works.
```
select to_number('USD34234.4350', 'L99999.0000');  // 34234.435
select to_number('EUR34234.4350', 'L99999.0000');  // 34234.435
select to_number('RY34234.4350', 'L99999.0000');  // 34234.435
```
Although 'RY ' is not a valid locale , the result is the same as the others .

Reply:
I think the description of locales is not consistent with the behavior in fact.